### DISCUSSION TITLE
v12.324.0

### DISCUSSION THREAD

**Discussion by felddy** [felddy/foundryvtt-docker#976] [General]
This container supports Foundry VTT stable release `12.324`

Please read the official FoundryVTT release notes for details on how to safely migrate to version `12`.

See:
- https://foundryvtt.com/releases/12.324

> [!WARNING]  
> While this is categorized as a stable release there is always a possibility of unexpected bugs or compatibility issues. As with any time you update the core software, be sure to perform a complete backup of your user data to minimize any risk of data loss.

## What's Changed
* Bump actions/cache from 3.3.2 to 4.0.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/880
* Bump actions/dependency-review-action from 3.1.5 to 4.0.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/881
* Update to use new reusable workflows by @felddy in https://github.com/felddy/foundryvtt-docker/pull/883
* Bump actions/download-artifact from 3.0.2 to 4.1.1 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/875
* Bump actions/upload-artifact from 4.0.0 to 4.2.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/882
* Bump step-security/harden-runner from 2.6.1 to 2.7.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/888
* Bump actions/upload-artifact from 4.2.0 to 4.3.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/884
* Remove `lazy-fixture-pytest` library for upgrade to `pytest` `v8.0.0` by @felddy in https://github.com/felddy/foundryvtt-docker/pull/889
* Bump fetch-cookie from 2.1.0 to 2.2.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/890
* Bump actions/download-artifact from 4.1.1 to 4.1.2 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/892
* Bump actions/upload-artifact from 4.3.0 to 4.3.1 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/893
* Bump setuptools from 69.0.3 to 69.1.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/895
* Bump pre-commit from 3.6.0 to 3.6.1 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/894
* Bump pytest from 8.0.0 to 8.0.2 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/904
* Bump setuptools from 69.1.0 to 69.1.1 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/903
* Bump pre-commit from 3.6.1 to 3.6.2 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/897
* Bump actions/download-artifact from 4.1.2 to 4.1.3 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/906
* Bump actions/dependency-review-action from 4.0.0 to 4.1.3 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/900
* Bump fetch-cookie from 2.2.0 to 3.0.1 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/905
* Bump actions/cache from 4.0.0 to 4.0.1 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/907
* Bump actions/download-artifact from 4.1.3 to 4.1.4 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/910
* Bump pytest from 8.0.2 to 8.1.1 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/913
* Bump typescript from 5.3.3 to 5.4.2 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/912
* Fix deprecated logging call by @felddy in https://github.com/felddy/foundryvtt-docker/pull/915
* Bump winston from 3.11.0 to 3.12.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/909
* Bump wheel from 0.42.0 to 0.43.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/914
* Fix build status badge by @felddy in https://github.com/felddy/foundryvtt-docker/pull/916
* Bump actions/checkout from 4.1.1 to 4.1.2 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/917
* Bump setuptools from 69.1.1 to 69.2.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/918
* Add hotfix for issue #697 - v11 database glibc workaround by @felddy in https://github.com/felddy/foundryvtt-docker/pull/919
* Add cookbooks and instructions for running FoundryVTT behind a Cloudflare Tunnel by @felddy in https://github.com/felddy/foundryvtt-docker/pull/920
* Bump typescript from 5.4.2 to 5.4.3 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/925
* Bump winston from 3.12.0 to 3.13.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/929
* Bump actions/cache from 4.0.1 to 4.0.2 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/923
* Bump actions/dependency-review-action from 4.1.3 to 4.2.4 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/926
* Bump pre-commit from 3.6.2 to 3.7.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/927
* Bump actions/dependency-review-action from 4.2.4 to 4.2.5 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/930
* Bump actions/setup-python from 5.0.0 to 5.1.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/931
* Bump typescript from 5.4.3 to 5.4.5 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/934
* Bump pytest-cov from 4.1.0 to 5.0.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/928
* Bump setuptools from 69.2.0 to 69.5.1 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/936
* Bump actions/checkout from 4.1.2 to 4.1.3 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/941
* Bump actions/download-artifact from 4.1.4 to 4.1.5 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/939
* Bump actions/upload-artifact from 4.3.1 to 4.3.2 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/938
* Bump actions/upload-artifact from 4.3.2 to 4.3.3 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/943
* Bump actions/download-artifact from 4.1.5 to 4.1.6 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/942
* Bump pytest from 8.1.1 to 8.2.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/950
* Bump actions/checkout from 4.1.3 to 4.1.4 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/947
* Bump actions/download-artifact from 4.1.6 to 4.1.7 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/946
* Testing release 12.320.0 by @felddy in https://github.com/felddy/foundryvtt-docker/pull/951
* Revert "Testing release 12.320.0" by @felddy in https://github.com/felddy/foundryvtt-docker/pull/952
* Bump coveralls from 3.3.1 to 4.0.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/957
* Bump step-security/harden-runner from 2.7.0 to 2.7.1 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/956
* Bump actions/dependency-review-action from 4.2.5 to 4.3.1 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/955
* Bump coverage from 6.5.0 to 7.5.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/944
* Bump actions/dependency-review-action from 4.3.1 to 4.3.2 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/958
* Bump actions/checkout from 4.1.4 to 4.1.5 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/960
* Bump ossf/scorecard-action from 2.3.1 to 2.3.3 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/961
* Bump pre-commit from 3.7.0 to 3.7.1 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/962
* Bump coveralls from 4.0.0 to 4.0.1 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/964
* Bump actions/checkout from 4.1.5 to 4.1.6 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/965
* Bump pytest from 8.2.0 to 8.2.1 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/966
* Pin requests dependency to work around docker-py incompatibility by @felddy in https://github.com/felddy/foundryvtt-docker/pull/972
* Bump setuptools from 69.5.1 to 70.0.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/971
* Bump step-security/harden-runner from 2.7.1 to 2.8.0 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/970
* Bump github/codeql-action from 2.13.4 to 3.25.6 by @dependabot in https://github.com/felddy/foundryvtt-docker/pull/968
* Unpin requests and bump docker-py by @felddy in https://github.com/felddy/foundryvtt-docker/pull/974
* Bump FoundryVTT version to 12.324 by @felddy in https://github.com/felddy/foundryvtt-docker/pull/975

**Full Changelog**: https://github.com/felddy/foundryvtt-docker/compare/v11.315.1...v12.324.0

<hr /><em>This discussion was created from the release <a href='https://github.com/felddy/foundryvtt-docker/releases/tag/v12.324.0'>v12.324.0</a>.</em>

**Comment by tobenamed** [comment]
When you use the docker command to update,  when you use use 12.314.0 as the repo suffix, it says this:
![image](https://github.com/felddy/foundryvtt-docker/assets/19497518/6d18d869-dd0b-43d6-a9d4-7a1338fb92f5)

this is the docker-compose.yml:
![image](https://github.com/felddy/foundryvtt-docker/assets/19497518/799acc0a-7ca5-41cd-8baf-e3ddb4ba8d38)
